src/Mod/Sketcher/App/SketchObject.h

```diff
@@ -46,6 +46,20 @@ namespace Sketcher
 
 class SketchAnalysis;
 
+struct ExternalToAdd
+{
+    App::DocumentObject* obj;
+    std::string subname;
+    bool defining;
+    bool intersection;
+};
+enum class ExtTypes

```

Comment:
This enum should be really named `ExtType`, without s. It is important because when you specify the function arguments you have `ExtTypes type` and type being plural suggests that it is some kind of list, which it isn't.

Reply:
Fair enough. Updated